ocs_ci/ocs/resources/pod.py

```diff
@@ -300,6 +300,55 @@ def run_git_clone(self):
         assert wl.setup(), "Setup up for git failed"
         wl.run()
 
+    def install_packages(self, packages):
+        """
+        Install packages in a Pod
+
+        Args:
+            packages (list): List of packages to install
+
+        """
+        if isinstance(packages, list):
+            packages = ' '.join(packages)
+
+        cmd = f"yum install {packages} -y"
+        self.exec_cmd_on_pod(cmd, False)
+
+    def copy_to_node(self, server, authkey, localpath, remotepath, user=None):
```

Comment:
copy_to_server cause it's not the node yet when you call this function.

Reply:
Done